From 12705ba67abcc436e0923d63134d37aa0178590e Mon Sep 17 00:00:00 2001
From: Daniel Kulp <dan@kulp.com>
Date: Sun, 9 Aug 2026 13:50:05 -0400
Subject: [PATCH] bone-pinmux-helper: drop the <linux/of_gpio.h> include

Upstream removed the header in b6420bd5aa0c ("gpio: remove
of_get_named_gpio() and <linux/of_gpio.h>"), so the driver no longer
compiles on 7.1.

The include was already dead weight: bone-pinmux-helper.c is pure
pinctrl and references no symbol from that header -- no
of_get_named_gpio(), no of_gpio_* helper, nothing. Just drop it.
---
 drivers/misc/cape/beaglebone/bone-pinmux-helper.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/cape/beaglebone/bone-pinmux-helper.c b/drivers/misc/cape/beaglebone/bone-pinmux-helper.c
index 11c271d2cdd0..72093a7e89cc 100755
--- a/drivers/misc/cape/beaglebone/bone-pinmux-helper.c
+++ b/drivers/misc/cape/beaglebone/bone-pinmux-helper.c
@@ -27,7 +27,6 @@
 #include <linux/err.h>
 #include <linux/of.h>
 #include <linux/of_device.h>
-#include <linux/of_gpio.h>
 #include <linux/slab.h>
 #include <linux/pinctrl/pinctrl.h>
 #include <linux/pinctrl/pinmux.h>
-- 
2.43.0

